docs/user-guide/groupby.ipynb

```diff
@@ -195,6 +193,47 @@
     "grouped = sc.groupby(data, group=param, bins=bins).mean('x') # note the lack of quotes around param!\n",
     "sc.plot(grouped)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Usage examples\n",
+    "\n",
+    "### Filtering a variable using `groupby.copy`\n",
+    "\n",
+    "Apart from reduction operations discussed above, `groupby` also supports `copy`, which allows us to extract a group without changes.\n",
+    "We can use this, e.g., to filter data.\n",
+    "This can be used for filtering variables:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "var = sc.array(dims=['x'], values=np.random.rand(100))\n",
+    "select = var < 0.5 * sc.Unit('')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We use a temporary data array pass `1` to `copy` to extract the second group (group indices start at 0).\n",
```

Comment:
I know you are saying that is it to select the second group, but I am wondering if it could be interpreted as being slightly ambiguous what the second group means. Are you saying you are selecting the values where the condition `select` is `True` (i.e. `True` is 1 as an integer)? or are you saying you have 2 groups and 1 is the index in the list of groups.

Is there a way to make this slightly less ambuguous?
Or maybe I am worrying too much..